client/landing/gutenboarding/components/domain-picker/button.tsx

```diff
@@ -71,15 +68,15 @@ const DomainPickerButton: FunctionComponent = () => {
 				className={ classnames( 'domain-picker__button', { 'is-open': isDomainPopoverVisible } ) }
 				onClick={ () => setDomainPopoverVisibility( s => ! s ) }
 			>
-				{ head( freeDomainSuggestions )?.domain_name ?? '\u00a0' }
+				Choose a domain
```

Comment:
Needs i18n

```suggestion
				NO__( 'Choose a domain' )
```

But it's not really reflecting the mockup we're implementing :thinking: 

May be fine as a stop-gap before #37820 :thinking: